integ/components/deadline/deadline_05_secretsManagement/test/deadline_05_secretsManagement.test.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as CloudFormation from 'aws-sdk/clients/cloudformation';
+import * as SecretsManager from 'aws-sdk/clients/secretsmanager';
+import * as AWS from 'aws-sdk/global';
+import awaitSsmCommand from '../../common/functions/awaitSsmCommand';
+
+// Name of testing stack is derived from env variable to ensure uniqueness
+const testingStackName = 'RFDKInteg-SM-TestingTier' + process.env.INTEG_STACK_TAG?.toString();
+
+const cloudformation = new CloudFormation();
+const secretsManager = new SecretsManager();
+
+const bastionRegex = /bastionId/;
+const smSecretRegex = /deadlineSecretsManagementCredentialsSM(\d)/;
+
+const testCases: Array<Array<any>> = [
+  [ 'Secrets Management is enabled', 1 ],
+];
+let bastionId: string;
+let smSecretArns: Array<any> = [];
+
+beforeAll(async () => {
+  // Query the TestingStack and await its outputs to use as test inputs
+  const describeStacksResponse = await cloudformation.describeStacks({ StackName: testingStackName }).promise();
+  let stackOutput = describeStacksResponse.Stacks![0].Outputs!;
+  stackOutput.forEach(output => {
+    let outputKey = output.OutputKey!;
+    let outputValue = output.OutputValue!;
+
+    if (bastionRegex.test(outputKey)) {
+      bastionId = outputValue;
+    } else if (smSecretRegex.test(outputKey)) {
+      let testId = smSecretRegex.exec(outputKey)![1];
+      smSecretArns[+testId] = outputValue;
+    }
+  });
+});
```

Comment:
Is it worth being defensive here? After the loop, we could validate that we found the required outputs and failing if not.

Reply:
Good idea, added this in